renderdoc/driver/d3d11/d3d11_common.h

```diff
@@ -195,7 +195,17 @@ inline const char *GetDebugName(T *pObj)
   {
     HRESULT hr = pObj->GetPrivateData(WKPDID_D3DDebugObjectName, &size, tmpBuf);
     if(FAILED(hr))
-      return "";
+    {
+      static wchar_t tmpBufW[1024] = {0};
+      UINT wsize = size * 2;
```

Comment:
Maybe better written as `1023 * sizeof(wchar_t)` ? I'm not sure if `GetPrivateData` will modify `size` or not when it fails, best to avoid problems in case it does.